src/Planner/Planner.cpp

```diff
@@ -1081,7 +1081,9 @@ void addBuildSubqueriesForSetsStepIfNeeded(
     for (auto & subquery : subqueries)
     {
         auto query_tree = subquery->detachQueryTree();
-        auto subquery_options = select_query_options.subquery();
+        /// I suppose it should be better to use all flags from select_query_options,
+        /// But for now it is done in the same way as in old analyzer.
```

Comment:
What was the difference? It can affect the performance because it can for example reset `only_analyze`, can't it?

Reply:
only_analyze should be fine, cause subqueries won't be actually executed

Reply:
Could you please also clarify, what it affects?
(you can write directly in the code)